.travis.yml

```diff
@@ -1,21 +0,0 @@
-language: python
-cache: pip
```

Comment:
You might be losing the pip cache, but if the actions are running faster now anyways, probably doesn't matter.

Reply:
Yeah, the current CI finishes in a minute or two.